x-pack/plugins/observability_solution/observability_ai_assistant/server/functions/kibana.ts

```diff
@@ -67,7 +63,7 @@ export function registerKibanaFunction({
           '/internal/observability_ai_assistant/chat/complete',
           pathname
         ),
-        query,
+        query: query ? (query as Record<string, string>) : undefined,
```

Comment:
Why this change? Afaict this doesn't have any runtime implications and the type is already `Record<string, string> | undefined`

Reply:
will have a look at this as well, gracias